README.md

```diff
@@ -0,0 +1,3 @@
+# Skynet server #
+
+Developers should take a look at the [documentation](https://github.com/skynet-im/skynet-server/tree/master/docs).
```

Comment:
Why don't we use GitHub's Wiki function?

Reply:
There are ways around this problem, but they are unnecessarily complicated. I agree, we can leave it in the `docs`.